apps/website/netlify/edge-functions/homepage-redirect.ts

```diff
@@ -1,13 +1,12 @@
-// @ts-ignore
-import Negotiator from 'https://deno.land/x/negotiator/mod.ts';
+import Negotiator from 'https://deno.land/x/negotiator@1.0.1/mod.ts';
+import { Config } from 'https://edge.netlify.com';
```

Comment:
What are the best practices for Deno imports? Should we always pin a certain version?

Or `https://edge.netlify.com` is a special case and it should always be the latest?

Reply:
I think in that case it should be latest, because in the netlify runtime we also have no control over the version.